apps/confessions-server/src/util/auth.ts

```diff
@@ -0,0 +1,36 @@
+import { Request, Response, NextFunction } from "express";
+import { verify, JwtPayload } from "jsonwebtoken";
+import { IS_PROD } from "./isProd";
+
+export const ACCESS_TOKEN_SECRET = process.env.ACCESS_TOKEN_SECRET || "secrect";
```

Comment:
`secrect` lol

Reply:
the 'r's and 'c's do not work very well on my keyboard of this very old laptop. it is a miracle what I type compiles most of the time.